src/qibolab/platform.py

```diff
@@ -109,6 +109,9 @@ def setup(self):
         for qubit in self.qubits.values():
             if qubit.flux is not None and qubit.sweetspot != 0:
                 qubit.flux.offset = qubit.sweetspot
+        for coupler in self.couplers.values():
+            if coupler.flux is not None and coupler.sweetspot != 0:
+                coupler.flux.offset = coupler.sweetspot
```

Comment:
This will break all platforms that don't have couplers because `self.couplers` is `None` by default, so you cannot loop over it. I believe this is what is causing tests to fail. You should either check `if self.couplers is not None` before doing this loop, or use `field(default_factory=dict)` when initializing the couplers so that we can always loop over it.

There is a similar issue with the chevron routine in qibocal main, because you added https://github.com/qiboteam/qibocal/blob/7893a8c052377ed5467d4f7bc4813ca8bd6941e0/src/qibocal/protocols/characterization/two_qubit_interaction/chevron.py#L127 
which fails for platforms without couplers.

Reply:
Great, thanks !